### DISCUSSION TITLE
[Existing Solutions] Main Page Wireframe selection

### DISCUSSION THREAD

**Discussion by JeremySanchez1** [uprm-inso4116-2023-2024-S1/semester-project-college-toolbox#62] [Polls]
### **PAGE_1:** 
![WhatsApp Image 2023-09-21 at 19 45 17](https://github.com/uprm-inso4116-2023-2024-S1/semester-project-college-toolbox/assets/71047020/c868c00d-6105-443e-80da-236d40d7fa4f)

### **PAGE_2:**
![WhatsApp Image 2023-09-21 at 19 45 18](https://github.com/uprm-inso4116-2023-2024-S1/semester-project-college-toolbox/assets/71047020/ad411fd2-89ed-4900-b6ce-9d5adaef8a73)

Page_1 would have filters on top. the filters  are based on the type that the app applies to. One app can have many types. They would work as sort of a drop-down for each section that the user may be interest in>

Each app will be separated by a grid, where each grid contains the image of the app, tags that they fall into and two links. One to send them to the page of the app and the other to show the description of the app that is contained in our db. This could be by expanding the text box but could be implemented in some other way.

Page_2 is very similar in most components
The filters would be on the side instead to not lose track of them so easily when scrolling. And instead of showing the tags, we can show a description and the tags be a "background" implementation with the filters. Basically not explicitly showing what tags an app falls into. We can change this if needed. Contrary to Page_1, this only has one link which directs the user to the apps page.

- @DevSebastian1

**Comment by EdwinC1339** [comment]
Love the PAGE_2 layout, just as clean as the first but adaptable to more properties if needed. Only observation is that it might be a little tricky to make this reactive to different screen sizes, if the menu and cards are displayed sequentially the user would need to scroll up and down on their phone for example. But I think some solution could be worked out.
